.gitignore

```diff
@@ -1,6 +1,7 @@
 # Keys
 Pear/Supporting/GoogleService-Info.plist
 Pear/Supporting/Keys.plist
+Pear/Supporting/Secrets.plist
```

Comment:
Maybe we can consolidate all out `.plists` into one? Gets confusing keeping track of too many `.plists` because we have to manually add to our repo each time when someone clones. If nothing else, it'd be great if we can rename `Secrets.plist`to something more descriptive.

Reply:
Agreed. The difference between secrets and keys is not obvious to me.